flight/Modules/VtolPathFollower/vtol_follower_fsm.c

```diff
@@ -671,8 +671,11 @@ static void go_enable_fly_home()
 		vtol_fsm_path_desired.End[2] = -RTH_MIN_ALTITUDE;
 	}
 
-	vtol_fsm_path_desired.StartingVelocity = RTH_VELOCITY;
-	vtol_fsm_path_desired.EndingVelocity = RTH_VELOCITY;
+	float rth_vel = 3.0f;
```

Comment:
Fair enough. Could add a comment-- either here or in the git commit message-- as to why this is set to ```3.0f```?